src/app/modules/webhook/webhook.validation.ts

```diff
@@ -28,6 +28,13 @@ export const validateWebhookUrl = (webhookUrl: string): Promise<void> => {
         ),
       )
     }
+    if (webhookUrlParsed.hostname.endsWith(appUrlParsed.hostname)) {
```

Comment:
the hostname is `form.gov.sg` right, so technically this should be `'.' + appUrlParsed.hostname` right? plus (nit) we can probably merge this and the above block with a `||`, and just use a common error message

Reply:
that is correct, thanks for the catch! I've made edits